src/main/webapp/createEvent.js

```diff
@@ -0,0 +1,30 @@
+//File to add all needed js code for the createEvent.html
+
+/**
+ * Formats form input:
+ *      Converts date to epoch value.
+ *      Imposes a 2 fix decimal value for the price.
+ * 
+ * @return none.
+ */
+function formatInput(){
+    /**
+     * @type {Object inputHTMLElement} date
+     * @type {Object inputHTMLElement} price
+     * @type {int} decimalIndex - Index at which the first dot('.') char appears
+     *                            in the price's value.
+     */
+    date = document.querySelector("form#new-event input[name='date']");
+    price = document.querySelector("form#new-event span input[name='price']")
+    decimalIndex = price.value.indexOf('.');
+
+    date.type = 'text'; //datatime-local type rejects the epoch value.
+    date.value = Date.parse(date.value); //Converts to epoch.
+
+    if(decimalIndex == -1){
```

Comment:
Can you explain what decimalIndex is and why we're checking if it's -1?

Reply:
`decimalIndex` is the position at which the first dot character is found in the string (price). If there is no dot character (meaning there is no decimal part) `indexOf()` returns -1. Creating the need to add 2 decimal digits (.00). The other if checks that the price does not have more than 2 decimals if so it rounds it. Java Throws an error when converting a string like "20" to a double data type.

Reply:
Now that I think about it I didn't test when the user inputs a price like "20."; I'll check that tomorrow.